bin/varnishd/cache/cache_vrt_vcl.c

```diff
@@ -138,22 +115,54 @@ VCL_Ref(struct vcl *vcl, struct worker *wrk)
 	Lck_Unlock(&vcl_mtx);
 }
 
+/*
+ * We do not garbage collect discarded VCL's here, that happens in VCL_Poll()
+ * which is called from the CLI thread.
+ */
+
+#define vcl_rel(vcl) do {			\
+	assert((vcl)->busy > 0);		\
+	(vcl)->busy--;				\
+	(vcl) = NULL;				\
+	} while (0)
```

Comment:
ws ocd

Also, why not a static inline function for readability?

Reply:
* It is used under `vcl_mtx`
* "autogen.des" builds with inlining disabled